### ISSUE TITLE
[Markup and meta data] HTML markup isn't valid. (09.01.1)

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#34874]
#### VFS team name
Claims & Appeals

#### VFS product name
HLR V2

#### Point of Contact/Reviewers
Sarah Koomson (@Saraharaba8) - Accessibility

---

### Platform Issue
HTML markup isn't valid.

### Issue Details
Menu dropdowns should be tagged for assistive technology

### VA.gov Experience Standard
[Category Number 09, Issue Number 01](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/VA.gov-experience-standards.1683980311.html)

### Other References
WCAG SC 1.3.1_A

---

### Platform Recommendation
Menu items that have a dropdown menu include an "aria-haspopup" attribute that equals to "true".

### VFS Team Tasks to Complete

- [ ] Comment on the ticket if there are questions or concerns
- [ ] Close the ticket when the issue has been resolved or validated by your Product Owner. If a team has additional questions or needs Platform help validating the issue, please comment in the ticket.

**Comment by Mottie** [comment]
I spoke with @Saraharaba8, and this ticket is regarding the mega menu in the page header. The site-wide team owns this and reported that it will be replaced soon.

**Comment by shiragoodman** [comment]
FYI @brianalloyd I've added your teams label after reviewing @Mottie's comment above and chatting with Sitewide Content. If you disagree, please let me know. As always, please don't close this ticket until the issue has been addressed. Thanks!

**Comment by joshkimux** [comment]
@Saraharaba8 removing the `hlr` label as this is specific to sitewide 🙏

**Comment by shiragoodman** [comment]
@joshkimux is there a sitewide label we can use? We require a product label on all issue tickets, so if HLR is removed, we need one to replace it with.

**Comment by joshkimux** [comment]
Attached sitewide here!

**Comment by FranECross** [comment]
@laflannery With the new header/footer work being done, is this ticket obsolete, since the a11y review and work will be done during the new build? Thanks!

**Comment by FranECross** [comment]
I talked to Laura regarding this. We hope to get it updated in the next iteration of the header, which will more than likely be when the Design Team componentizes it. @FranECross to ping Matt Dingee when they start that work. Moving to Transfer to Other Teams

**Comment by laflannery** [comment]
After looking more into this and the aria standards, `aria-haspopup="menu"` should really only be used when the element popping up has been marked with the role of menu. That isn't the case here, the auth menu is just a list of links so I am not recommending that this be updated to included this aria attribute
